Source/WebCore/crypto/cocoa/CryptoUtilitiesCocoa.cpp

```diff
@@ -146,6 +146,7 @@ Vector<uint8_t> calculateHMACSignature(CCHmacAlgorithm algorithm, const Vector<u
         digestLength = CC_SHA1_DIGEST_LENGTH;
         break;
     case kCCHmacAlgSHA224:
+        RELEASE_ASSERT_NOT_REACHED_WITH_MESSAGE(sha224DeprecationMessage);
         digestLength = CC_SHA224_DIGEST_LENGTH;
```

Comment:
Should `CC_SHA224_DIGEST_LENGTH` be renamed to `DEPRECATED_CC_SHA224_DIGEST_LENGTH`?

Reply:
That is a commoncrypto enum which i can't change. I will use 256 here to just follow that convention.